### PR TITLE
Add dash-labs via grayskull

### PR DESCRIPTION
**Pull request by asford**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by asford** [comment]
@conda-forge/help-python This is ready for review.

File: recipes/dash-labs/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  version: {{ version }}
+
+source:
+  # v0.2.0 sdist on pypi is invalid and doesn't include requirements.txt
+  # Install from github archive
+  url: https://github.com/plotly/dash-labs/archive/v{{ version }}.tar.gz
+  sha256: 7f7b77881476059206e45e38360afe34ab3b5ff2e2ca94a90ffd6ae239b1970c
+build:
+  number: 0
+  skip: true   # [py<36]
```
Comment (mariusvniekerk) [inline recipes/dash-labs/meta.yaml:16]:
```suggestion
  noarch: python
```

File: recipes/dash-labs/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python
```
Comment (mariusvniekerk) [inline recipes/dash-labs/meta.yaml:22]:
```suggestion
    - python >=3.6
```

File: recipes/dash-labs/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+  run:
+    - dash >=1.18.1
+    - dash-bootstrap-components >=0.11.1
+    - dataclasses ==0.8  # [py<37]
```
Comment (mariusvniekerk) [inline recipes/dash-labs/meta.yaml:26]: Conda forge has a noop version of dataclasses for python37+
```suggestion
    - dataclasses ==0.8 
```

File: recipes/dash-labs/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+    - python
```
Comment (mariusvniekerk) [inline recipes/dash-labs/meta.yaml:27]:
```suggestion
    - python >=3.6
```

File: recipes/dash-labs/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,45 @@
...
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - dash >=1.18.1
+    - dash-bootstrap-components >=0.11.1
+    - dataclasses ==0.8
```
Comment (asford) [inline recipes/dash-labs/meta.yaml:26]:
```suggestion
    - dataclasses >=0.8 
```

Relaxing pin

**Comment by asford** [comment]
Fantastic, thank you. 🙏

**Pull request merged**